include/ftxui/screen/color.hpp

```diff
@@ -2,7 +2,7 @@
 #define FTXUI_SCREEN_COLOR
 
 #include <stdint.h>  // for uint8_t
-#include <string>    // for wstring
```

Comment:
Could you bring back the spaces?

Reply:
(Ooops, I merged it. I will do it later myself)

Reply:
I'm sorry; I completely overlooked that. I hate unnecessary whitespace changes 😓